.travis.yml

```diff
@@ -1,50 +1,67 @@
-sudo: required
+# Variables needed for this script are stored on Travis Settings
+#  - For all environments:
+#    - AWS_DEFAULT_REGION
+#  - For dev environment:
+#    - AWS_ACCOUNT_USER_ID_DEV
+#    - AWS_ACCOUNT_SECRET_DEV
+#    - AWS_PROFILE_DEV
+#  - For Demo environment:
+#   - AWS_ACCOUNT_USER_ID_DEMO
+#   - AWS_ACCOUNT_SECRET_DEMO
+#   - AWS_PROFILE_DEMO
+# - For Prod environment:
+#   - AWS_ACCOUNT_USER_ID_PROD
+#   - AWS_ACCOUNT_SECRET_PROD
+#   - AWS_PROFILE_PROD
+
 language: go
 
-before_install:
-  - curl -sf https://up.apex.sh/install | sudo sh
-  - sudo up upgrade
+go:
+ - 1.13.x
 
-services:
-  - docker
+before_install:
+ - curl -sf https://up.apex.sh/install | sudo sh
+ - pip install --user awscli
+ - export PATH=$PATH:$HOME/.local/bin
 
 install:
-  - go get -t ./...
-  - sudo apt-get install jq -y
-
-after_success:
-  - test -n "$TRAVIS_TAG" && docker login -u=uneetci -p="$DOCKER_PASSWORD"
+ - go get -t
+ - sudo apt-get install jq -y
 
 script:
-  - go test -v ./...
+ # We configure a specific AWS profile for testing
+ # This will ONLY test the DEV configuration at this stage!
+ - aws configure set profile.travis-test-dev.aws_access_key_id $AWS_ACCOUNT_USER_ID_DEV
+ - aws configure set profile.travis-test-dev.aws_secret_access_key $AWS_ACCOUNT_SECRET_DEV
+ - aws configure set profile.travis-test-dev.region $AWS_DEFAULT_REGION
+ - aws configure --profile travis-test-dev list
+ # We run the script that will define the environment variables we need for the test
+ - export TRAVIS_AWS_PROFILE=travis-test-dev
+ - ./aws.env
```

Comment:
How about a test after here to check the env is actually exported?

Reply:
This is a good idea, I have one already in theory in the `aws.env` script itself but we can probably get something more robust...
Not sure how to do that yet, I'll have to research that.